packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -39,9 +44,29 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
     @property({ type: Boolean })
     public hasPrimaryActionsOnly = false;
 
+    @property({ type: Object })
+    public locale:CookieBannerLocale = defaultLocale;
+
     @queryAll('pie-toggle-switch')
         _preferencesNodes!: NodeListOf<PieToggleSwitch>;
 
+    private _customTagEnhancers:CustomTagEnhancers = {
+        linkStatement: (tagContent:string) => html`<pie-link variant="inverse">${tagContent}</pie-link>`,
+        linkNecessaryOnly: (tagContent:string) => html`<pie-link data-test-id="body-necessary-only" tag="button" variant="inverse" @click="${this._onNecessaryOnly}">${tagContent}</pie-link>`,
+        linkManagePreferences: (tagContent:string) => html`<pie-link data-test-id="body-manage-prefs" tag="button" variant="inverse" @click="${this._openManagePreferencesModal}">${tagContent}</pie-link>`,
+        linkAcceptAll: (tagContent:string) => html`<pie-link data-test-id="body-accept-all" tag="button" variant="inverse" @click="${this._onAcceptAll}">${tagContent}</pie-link>`,
+        linkCookieStatement: (tagContent:string) => html`<pie-link href="#" size="small" target="_blank">${tagContent}</pie-link>`,
```

Comment:
this looks fantastic, I am curious how we will add `href` values to the links, which could differ between locales?

Reply:
@ashleynolan Do you know if the missing `hrefs` will be owned by us or passed as props?

Reply:
They'll be owned by us. We can likely fill these i ourselves, or ask Ellis to confirm the links with the countries as we roll them out.

We'll only need Denmark and Ireland right now though, so I imagine we could just add these manually.

IE = https://www.just-eat.ie/info/cookies-policy
DK = https://www.just-eat.dk/cookie-erklaering

One thing worth noting for Denmark though just from having a quick look, is that the URL changes dependent on the language specified on the site. So based on that, we'll likely need to set-up language files for `enDK`, so that we can handle those URLs. (I'm not sure how we'd do that otherwise? Might need a separate ticket to add the language JSON needed for Denmark and handle the URL additions there.

Reply:
@jamieomaguire Already raised this question in the #proj-cookieconsent Slack channel... 💭